frameworks/Java/proteus/conf/application.conf

```diff
@@ -26,7 +29,7 @@ globalHeaders
 #  Access-Control-Allow-Origin: "*"
 #  Access-Control-Allow-Methods: "*"
 #  Access-Control-Allow-Headers: "*"
-  Server: ${application.name}
+  Server: "P"
```

Comment:
Is the server name really 'P'? This seems like a cheap attempt at dropping a few bytes from every request.